src/resource-metadata/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * AWS Resource Tags Lambda function for Coralogix
+ *
+ * @file        This file is lambda function source code
+ * @author      Coralogix Ltd. <info@coralogix.com>
+ * @link        https://coralogix.com/
+ * @copyright   Coralogix Ltd.
+ * @licence     Apache-2.0
+ * @version     0.1.0
+ * @since       0.1.0
+ */
+
+"use strict";
+
+const assert = require("assert");
+const protoLoader = require("@grpc/proto-loader");
+const grpc = require("@grpc/grpc-js");
+const util = require('util');
+
+const { collectLambdaResources } = require("./lambda.js")
+const { collectEc2Resources } = require("./ec2.js")
+
+// Check Lambda function parameters
+assert(process.env.PRIVATE_KEY, "No private key!")
+const privateKey = process.env.PRIVATE_KEY
+assert(process.env.CORALOGIX_METADATA_URL, "No Coralogix metadata URL key!")
+const coralogixMetadataUrl = process.env.CORALOGIX_METADATA_URL
+
```

Comment:
This can be extracted to another file.
```suggestion
const validateAndExtractConfiguration = () => {
    assert(process.env.PRIVATE_KEY, "No private key!")
    const privateKey = process.env.PRIVATE_KEY
    assert(process.env.CORALOGIX_METADATA_URL, "No Coralogix metadata URL key!")
    const coralogixMetadataUrl = process.env.CORALOGIX_METADATA_URL
    return {privateKey, coralogixMetadataUrl};
};

const {privateKey, coralogixMetadataUrl} = validateAndExtractConfiguration();

```